quizroyale/db/errors.py

```diff
@@ -0,0 +1,4 @@
+class NoSuchUserException(Exception):
+    def __init__(self):
```

Comment:
```suggestion
    def __init__(self) -> None:
```

Reply:
I don't think that typing init method has any sense